gcp/function/src/middlewares/redirectTrailingSlash.ts

```diff
@@ -0,0 +1,70 @@
+import type express from "express";
+
+import { THIRTY_DAYS } from "../constants.js";
+import { VALID_LOCALES } from "../internal/constants/index.js";
+import { redirect } from "../utils.js";
+
+// Note that the keys of "VALID_LOCALES" are lowercase locales.
+const LOCALE_URI_WITHOUT_TRAILING_SLASH = new Set(
+  [...VALID_LOCALES.keys()].map((locale) => `/${locale}`)
+);
+const LOCALE_URI_WITH_TRAILING_SLASH = new Set(
+  [...VALID_LOCALES.keys()].map((locale) => `/${locale}/`)
+);
+// TODO: The code that uses LEGACY_URI_NEEDING_TRAILING_SLASH should be
+//       temporary. For example, when we have moved to the Yari-built
+//       account settings page, we should add fundamental redirects
+//       for "/{locale}/account/?" and "/account/?" that redirect to
+//       "/{locale}/settings" and "/settings" respectively. The other
+//       cases can be either redirected or deleted eventually as well.
+//       The goal is to eventually remove the code that uses
+//       LEGACY_URI_NEEDING_TRAILING_SLASH.
+const LEGACY_URI_NEEDING_TRAILING_SLASH = new RegExp(
+  `^(?:${[...LOCALE_URI_WITHOUT_TRAILING_SLASH].join(
+    "|"
+  )})?/(?:account|contribute|maintenance-mode|payments)/?$`
+);
```

Comment:
```suggestion
```

Reply:
not needed, `/accounts/` is handled in fundamental all others seem to be dependent on kuma.